front-end/src/components/Post/SecondProposal.tsx

```diff
@@ -14,30 +14,29 @@ interface Props {
 	proposalId?: number | null | undefined
 	api?: ApiPromise,
 	apiReady?: boolean,
-	getLinkedAccount: () => Promise<InjectedAccountWithMeta | undefined>
+	address: string
+	defaultAddress: string
+	addressOptions: DropdownItemProps[]
+	accounts: InjectedAccountWithMeta[]
+	onAccountChange: (event: React.SyntheticEvent<HTMLElement, Event>, data: DropdownProps) => void
+	getAccounts: () => Promise<undefined>
 }
 
-const SecondProposal = ({ className, api, apiReady, getLinkedAccount, proposalId }: Props) => {
+const SecondProposal = ({ className, api, apiReady, proposalId, address, defaultAddress, accounts, addressOptions, onAccountChange, getAccounts }: Props) => {
 	const { queueNotification } = useContext(NotificationContext);
 
 	const secondProposal = async () => {
 		if (!api) {
 			return;
```

Comment:
Can we log a console.error here, just in case?
Same below for the proposalId. This should never happen, but who knows..

Reply:
done